package-lock.json

```diff
@@ -887,7 +887,7 @@
     "minimatch": {
       "version": "3.0.4",
       "resolved": "https://registry.npmjs.org/minimatch/-/minimatch-3.0.4.tgz",
-      "integrity": "sha512-yJHVQEhyqPLUTgt9B83PXu6W3rx4MvvHvSUvToogpwoGDOUQ+yDrR0HRot+yOCdCO7u4hX3pWft6kWBBcqh0UA==",
+      "integrity": "sha1-UWbihkV/AzBgZL5Ul+jbsMPTIIM=",
```

Comment:
These changes are indicative of your machine getting out of sync with the dependencies installed on other dev machines. try a `git checkout cms-dep package-lock.json` `rm -f node_modules` `npm cache clean -f` and `npm install`